utide/utilities.py

```diff
@@ -302,8 +302,7 @@ def convert_unicode_arrays(b):
             if val.dtype.kind == 'O':
                 newval = np.empty(shape=val.shape, dtype=val.dtype)
                 for k, x in enumerate(val):
-                    if (isinstance(x, np.ndarray) and x.dtype.kind == 'U'
-                            and x.size == 1):
+                    if (isinstance(x, np.ndarray) and x.dtype.kind == 'U' and x.size == 1):  # noqa
```

Comment:
If we are going to move this to a single line anyway, I would prefer we remove the `()` at the beginning and the end.

We could remove the `# noqa` though if we just fix it. With the `()`, we should be able to split it over multiple lines (like we had it), but fix the indentation (cause to me the old indentation looks off)

Reply:
Fixed that in the latest commit. Thanks!